spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-stream-functional-binder-sample/README.adoc

```diff
@@ -23,8 +23,14 @@ The topic and subscription will be created in your account if they do not alread
 
 1. In separate terminal windows, start the `spring-cloud-gcp-pubsub-stream-functional-binder-sample-source` and `spring-cloud-gcp-pubsub-stream-functional-binder-sample-sink` applications with the `$ mvn spring-boot:run` command in the same folder as the apps' `pom.xml` files.
 
-5. Browse to the source application at `http://localhost:8080`, type in a message and a user name, and press the `Post it!` button.
+2. Browse to the source application at `http://localhost:8080`, type in a message and a user name, and press the `Post it!` button.
 
-6. Verify in the `spring-cloud-gcp-pubsub-stream-functional-binder-sample-sink` app's logs that the message was received:
+3. Verify in the `spring-cloud-gcp-pubsub-stream-functional-binder-sample-sink` app's logs that the message was received:
 +
 `New message received from [USERNAME] at [TIME_MESSAGE_SENT]: [MESSAGE]`
+
+
+== Cleanup
+
+Delete the test topic (`my-topic`) and subscription (`my-topic.my-group`).
```

Comment:
I feel like that if the sample creates these resources automatically, it should also clean them up automatically.

Reply:
This is problematic. There are two applications -- source and sink, which could both autocreate the named topic. If either app deletes the topic, the other app could still be running with the assumption that the topic exists.
I've stopped one of the two apps repeatedly as part of my own testing, and it was convenient that the other app ran without hiccups, waiting for me to start the first one again.